docs/use.md

```diff
@@ -0,0 +1,179 @@
+# Usage
+
+`cf create-service pubsub default foobar`
+
+`cf bind-service myapp foobar -c '{"role": "pubsub.admin"}'`
+
+Notes:
+
+`bind-service` calls require a role, except for Cloud SQL, Stackdriver Debugger, and Stackdriver Trace.
+`create-service` calls take the following optional custom parameters, all as strings.
+
+
+* [PubSub](https://cloud.google.com/pubsub/docs/)
+    * Provision
+        * `topic_name` (defaults to a generated value)
+        * `subscription_name`
+        * `is_push` (defaults to `false`, to set use "true")
+        * `endpoint` (for when is_push == "true", defaults to `nil`)
+        * `ack_deadline` (in seconds, defaults to `10`, max 600)
+    * Bind
+        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
+
+        **Example Binding credentials**
+
+        ```
+        "credentials": {
+             "Email": "redacted",
+             "Name": "redacted",
+             "PrivateKeyData": "redacted",
+             "UniqueId": "redacted",
+             "topic_name": "foobar",
+             "subscription_name": "empty_if_not_set",
+        }
+        ```
+
+* [Cloud Storage](https://cloud.google.com/storage/docs/)
+    * Provision
+        * `name` (defaults to a generated value)
+        * `location` (for options, see https://cloud.google.com/storage/docs/bucket-locations. Defaults to `"us"`)
+    * Bind
+        * `role` without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
```

Comment:
It would be nice to also note here when the role is or isn't required. In case someone missed the note about it above.